javascript/javascript1/week3/Note-taking-app.js

```diff
@@ -0,0 +1,40 @@
+const notes = [];
```

Comment:
Nice implementation of note handling! 😄 Be careful with the formatting though. There is a lot of additional space in this file that make it a bit harder to read. 🥺